inc/bpl_evm_api.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * TODO: Fill in file header, if necessary.
+ */
+
+#ifndef BPL_EVM_H
+#define BPL_EVM_H
+
+/******************************************************************************
+ INCLUDES
+ ******************************************************************************/
+
+#include "bplib_api_types.h"
+
+/************************************************
+ * Typedefs
+ ************************************************/
+
+typedef enum
+{
+    BPL_EVM_EventType_UNKNOWN = 0,
+    BPL_EVM_EventType_DEBUG = 1,
+    BPL_EVM_EventType_INFO = 2,
+    BPL_EVM_EventType_WARNING = 3,
+    BPL_EVM_EventType_ERROR = 4,
+    BPL_EVM_EventType_CRITICAL = 5
+} BPL_EVM_EventType_t;
+
+typedef struct
+{
+    uint32_t ReturnValue;
+} BPL_Status_t;
+
+#define BPL_STATUS_SUCCESS             (0u)
+#define BPL_STATUS_ERROR_GENERAL       (1u)
+#define BPL_STATUS_ERROR_INPUT_INVALID (2u)
+#define BPL_STATUS_ERROR_PROXY_INIT    (3u)
```

Comment:
I assume these should be moved to somewhere else (if we decide to keep the struct impl of the return types, as discussed).